RefrigeratorExe/RefrigeratorExe/Program.cs

```diff
@@ -0,0 +1,297 @@
+﻿namespace RefrigeratorExe
+{
+    internal class Program
+    {
+       
+        static void Main(string[] args)
+        {
+
+            bool flag = true;
+            int numberPress;
+            List<Refrigerator> refrigerators = new List<Refrigerator>();
+            Refrigerator refrigerator = new Refrigerator("Samsong","Red",4);
+            refrigerators.Add(refrigerator);
+            BuildRefrigerators(refrigerators);
+            Console.WriteLine("\nWelcome To The Refrigerator App");
+            
+            while (flag) 
+            {
+                Console.WriteLine("");
+                Console.WriteLine("Press 1 to print out Item in Refrigerator");
+                Console.WriteLine("Press 2 to print the amout of place left in Refrigerator");
+                Console.WriteLine("Press 3 to add item to Refrigeratoe");
+                Console.WriteLine("Press 4 to take out item our of Refrigerator");
+                Console.WriteLine("Press 5 to clean Refigerator");
+                Console.WriteLine("Press 6 What do you want to eat");
+                Console.WriteLine("Press 7 to sort item according to expiry date");
+                Console.WriteLine("Press 8 to sort Shelfs acccording to space");
+                Console.WriteLine("Press 9 to sort refrigerators according to space");
+                Console.WriteLine("Press 10 to get ready for shoping");
+                Console.WriteLine("Prss 100 to exit");
+                numberPress = InputNumber();
+                switch (numberPress)
```

Comment:
switch case is a bad practice. another way to implement it is via the dictionary I suggested you before